lerobot/configs/default.yaml

```diff
@@ -45,6 +42,9 @@ training:
   save_freq: ???
   num_workers: 4
   batch_size: ???
+  accelerate:
```

Comment:
Should we mention here that accelerate needs to be installed as an extra?

Reply:
Will mention it in the example in the `README`